docs/api/openapi3.json

```diff
@@ -3558,24 +3555,37 @@
             "content": {
               "application/json": {
                 "schema": {
-                  "$ref": "#/components/schemas/ListResponse_Group"
+                  "$ref": "#/components/schemas/Version"
                 }
               }
             },
             "description": "Success"
           }
         },
-        "summary": "ListUserGroups",
+        "summary": "Version",
         "tags": [
-          "Groups",
-          "Users"
+          "Misc"
         ]
       }
     },
-    "/api/version": {
+    "/v1/groups/{id}/grants": {
```

Comment:
shouldn't be in the api docs anymore. We could add support for redirects that change a path id to a query parameter to make supporting these easy.